api/auth_middleware_test.go

```diff
@@ -1,63 +1,101 @@
 package api_test
 
 import (
+	b64 "encoding/base64"
 	"fmt"
 	"net/http"
 	"testing"
 
+	"github.com/topfreegames/Will.IAM/models"
```

Comment:
nit: add a blank line, so we keep non-aliased and aliased imports in separate groups? (personal nit)